sql/parser/sql.y

```diff
@@ -2807,6 +2808,14 @@ func_expr_common_subexpr:
 | GREATEST '(' expr_list ')' { unimplemented() }
 | LEAST '(' expr_list ')' { unimplemented() }
 
+func_or_variable:
```

Comment:
we decided to kill the variable form for now since a functional form is consistent, and gels with a modern approach, and will punt changing this back to the standard variable form.